MediaBrowser.Controller/Entities/UserItemData.cs

```diff
@@ -19,17 +17,11 @@ public class UserItemData
         /// </summary>
         private double? _rating;
 
-        /// <summary>
-        /// Gets or sets the user id.
-        /// </summary>
-        /// <value>The user id.</value>
-        public Guid UserId { get; set; }
-
         /// <summary>
         /// Gets or sets the key.
         /// </summary>
         /// <value>The key.</value>
-        public string Key { get; set; }
+        public required string Key { get; set; }
```

Comment:
Slightly unrelated but it might be worth updating this summary because I have absolutely no idea what "the key" means